app/src/main/res/layout/activity_booking_detail.xml

```diff
@@ -394,6 +298,61 @@
                                 android:textSize="@dimen/sp_18"/>
                         </LinearLayout>
                     </LinearLayout>
+
+                    <LinearLayout
```

Comment:
WARNING: [DisableBaselineAlignment] Set `android:baselineAligned="false"` on this element for better performance
 When a LinearLayout is used to distribute the space proportionally between nested layouts, the baseline alignment property should be turned off to make the layout computation faster.